src/gdcdictionary/schemas/molecular_test.yaml

```diff
@@ -7098,9 +7247,22 @@ properties:
       - "_terms.yaml#/test_value_range/common"
     enum:
       - 0-25
+      - 1–49%
+      - 10–19%
+      - 20–29%
       - 26-50
```

Comment:
@bullenc , I'm going to group the whole numbers together and percentages together.  I foresee this being rare, so I don't think it would be all that necessary to add this to your formatting guide.

Reply:
Sounds good I'll remove this prop from being sorted in the script